tutorials/dataframe/df102_NanoAODDimuonAnalysis.C

```diff
@@ -75,39 +77,38 @@ void df102_NanoAODDimuonAnalysis()
                                  {"Muon_pt", "Muon_eta", "Muon_phi", "Muon_mass", "Muon_pair"});
 
    // Produce histogram of di-muon mass spectrum
-   auto h = df_mass.Histo1D({"Dimuon_mass", "Dimuon_mass", 20000, 0.25, 300}, "Dimuon_mass")
+   auto h = df_mass.Histo1D({"Dimuon_mass", "Dimuon_mass", 30000, 0.25, 300}, "Dimuon_mass")
                    .GetValue();
 
    // Make plot
    gStyle->SetOptStat(0);
    gStyle->SetTextFont(42);
-   auto c = new TCanvas("c", "c", 800, 600);
```

Comment:
It is not obvious but the leak is on purpose: we want the canvas and histo and the rest survive the scope in order for our doxy filter to inline the plot in the documentation page.

Reply:
Done!